cpp/ps0001/ps0001_sol.cpp

```diff
@@ -0,0 +1,20 @@
+#include <iostream>
+using namespace std;
+int main()
+{
+    // prompting user for 3 numbers to add 1/2
+    short num1, num2, num3;
+    cout << "please enter the first number" << endl;
+    cin >> num1;
+    cout << "please enter the second number" << endl;
+    cin >> num2;
+    cout << "please enter the third number" << endl;
+    cin >> num3;
+    
+    // calculating & outputting the result 3/3
+    short sum = num1 + num2 + num3;
+    cout << sum;
```

Comment:
- [x] did summation process for the three inputted numbers
- [x] used `std::cout` in the right way to display output from the summation process
- [ ] output is as required
```
155
940
168
--------
1263
```